website/content/api-docs/peering.mdx

```diff
@@ -0,0 +1,322 @@
+  the peering. This parameter is not required and does not directly impact the cluster
+  peering process.
+
+### Sample Payload
+
+```json
+{
+  "PeerName": "cluster-02",
+  "Meta": {
+    "env": "production"
+  }
+}
+```
+
+### Sample Request
+
+```shell-session
+$ curl --request POST \
+   --header "X-Consul-Token: 5cdcae6c-0cce-4210-86fe-5dff3b984a6e" \
+   --data @payload.json \
+   http://127.0.0.1:8500/v1/peering/token
+```
+
+### Sample Response
+
+```json
+{
+    "PeeringToken": "eyJDQSI6bnVsbCwiU2V..."
+}
+```
+
+## Establish a Peering
+
+This endpoint establishes a peering with a given peering token.
+
+| Method  | Path                 | Produces           |
+| ------- | -------------------- | ------------------ |
+| `POST`  | `/peering/establish` | `application/json` |
+
+This endpoint returns no data. Success or failure is indicated by the status
+code returned.
+
+The table below shows this endpoint's support for
+[blocking queries](/api-docs/features/blocking),
+[consistency modes](/api-docs/features/consistency),
+[agent caching](/api-docs/features/caching), and
+[required ACLs](/api#authentication).
+
+| Blocking Queries | Consistency Modes | Agent Caching | ACL Required     |
+| ---------------- | ----------------- | ------------- | ---------------- |
+| `NO`             | `none`            | `none`        | `none`           |
+
+### JSON Request Body Schema
+
+- `PeerName` `(string: <required>)` - The name assigned to the peer cluster.
+  The `PeerName` is used to reference the peer cluster in service discovery queries
+  and configuration entries such as `service-intentions`. This field must be a
+  valid DNS hostname label.
+
+- `Partition` `(string: "")` - <EnterpriseAlert inline /> The admin partition
+  that peers to the cluster that generated the peering token. Uses `default`
+  when not specified.
+
+- `PeeringToken` `(string: <required>)` - The peering token fetched from the
+  peer cluster.
+
+- `Datacenter` `(string: "")` - Specifies the datacenter. Defaults to the
+  agent's datacenter when not specified.
+
+- `Token` `(string: "")` - Specifies the ACL token to use in the request. Takes precedence
+  over the token specified in the `token` query parameter, `X-Consul-Token` request header,
+  and `CONSUL_HTTP_TOKEN` environment variable.
+
+- `Meta` `(map<string|string>: <optional>)` - Specifies KV metadata to associate with
+  the peering. This parameter is not required and does not directly impact the cluster
+  peering process.
+
+### Sample Payload
+
+```json
+{
+  "PeerName": "cluster-01",
+  "PeeringToken": "eyJDQSI6bnVsbCwiU2V...",
+  "Meta": {
+    "env": "production"
+  }
+}
+```
+
+### Sample Request
+
+```shell-session
+$ curl --request POST \
+   --header "X-Consul-Token: 5cdcae6c-0cce-4210-86fe-5dff3b984a6e" \
+   --data @payload.json \
+   http://127.0.0.1:8500/v1/peering/establish
+```
+
+### Sample Response
+
+```json
+{}
+```
+
+## Read a Peering
+
+This endpoint reads a peering with the given name.
```

Comment:
```suggestion
## Read a Peering Connection

This endpoint returns information about a peering connection for the specified peer name.
```
Trying to use 'peering' as a description of the connection rather than the object, itself, to keep the language neutral.